docs/docsite/rst/community/development_process.rst

```diff
@@ -35,20 +36,14 @@ pull request to backport the change to a previous stable branch.
 
 .. note::
 
-    These instructions assume that ``stable-2.5`` is the targeted release
-    branch for the backport.
-
-.. note::
-
-    These instructions assume that ``https://github.com/ansible/ansible.git``
-    is configured as a ``git remote`` named ``upstream``. If you do not use
-    a ``git remote`` named ``upstream``, adjust the instructions accordingly.
-
-.. note::
-
-   These instructions assume that ``https://github.com/<yourgithubaccount>/ansible.git``
-   is configured as a ``git remote`` named ``origin``. If you do not use
-   a ``git remote`` named ``origin``, adjust the instructions accordingly.
+    * These instructions assume that ``stable-2.5`` is the targeted release
+      branch for the backport.
+    * These instructions assume that ``https://github.com/ansible/ansible.git``
+      is configured as a ``git remote`` named ``upstream``. If you do not use
+      a ``git remote`` named ``upstream``, adjust the instructions accordingly.
+    * These instructions assume that ``https://github.com/<yourgithubaccount>/ansible.git``
+      is configured as a ``git remote`` named ``origin``. If you do not use
+      a ``git remote`` named ``origin``, adjust the instructions accordingly.
```

Comment:
Can we tighten this up to be a list of assumptions - something like:

These instructions assume that:
- stable-2.5 is the targeted . . . 
- ansible/ansible is . . 
- <yourgithubaccount>/ansible is . . .

Reply:
Done